tasks/terraform-cli/task.json

```
@@ -111,7 +112,7 @@
             "label": "Command Options",
             "helpMarkDown": "Any additional options for the selected command such as `-input=true`. Multiple options can be provided delimited by spaces as `-option1=value -option2=value -option3=value`",
             "required": "false",
-            "visibleRule": "command != forceunlock"
+            "visibleRule": "command != forceunlock && command != workspaceselect"

```

Shouldn't this be `&& command != workspace`? Or am I missing concatenation of command and subcommand somewhere?
```suggestion
            "visibleRule": "command != forceunlock && command != workspace"

```

---

fixed. not necessarily workspace but, the workspace subcommand. `workspace select` for example doesnt take any options. So the rule is based on the subcommand.